acceptance-tests.md

```diff
@@ -114,8 +134,10 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Establish OpenShift cluster upgrade process**
     * _As an administrator of the cluster, I should be able to follow a set of documented instructions to help me in upgrading to newer versions of OpenShift. The rule book should also establish the process to mitigate any issues that might arise during the upgrade._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. Upgrade documentation available in the documentation repository
+    * Acceptance tests:
```

Comment:
Are there any steps to upgrading that are unique to NERC and not documented in the general Red Hat product upgrade docs.  (There are, and that's what should be documented here, in addition to the standard Red Hat upgrade docs links.)